app/login/page.tsx

```diff
@@ -117,6 +116,7 @@ export default function Page({
                   type="checkbox"
                   className="peer absolute left-0 h-5 w-5 cursor-pointer appearance-none rounded-md border-2 border-blue-gray-200 transition-all checked:border-gray-900 checked:bg-gray-900"
                   id="check"
+                  onChange={(e) => setRememberMe(e.target.checked)} // Update rememberMe state on change
```

Comment:
You can make this an [uncontrolled component](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components), and add a name, e.g., "check" and access it from `formData`. It will have `'on'` when it is checked.